File: fizzbuzz.rs
pub fn fizzbuzz() {
	 for x in 0..100{
		 if x % 15 == 0 {
			println!("fizzbuzz")
		} else if x % 5 == 0 {
			println!("buzz")
		} else if x % 3 == 0 {
			println!("fizz")
		}
		else {
			println!("{}", x)
		}
	}
 
}

File: fizzbuzz.lua
function fizzbuzz(n) 
  if n % 3 == 0 and n % 5 == 0 then 
    return "FizzBuzz" 
  elseif n % 3 == 0 then 
    return "Fizz" 
  elseif n % 5 == 0 then 
    return "Buzz" 
  else 
    return tostring(n) 
  end 
end 
 
for i = 1, 100 do 
  print(fizzbuzz(i)) 
end 
